src/lib/viewers/iframe/__tests__/IFrameLoader-test.js

```diff
@@ -1,10 +1,23 @@
-/* eslint-disable no-unused-expressions */
 import IFrameLoader from '../IFrameLoader';
 import IFrameViewer from '../IFrameViewer';
 
 const sandbox = sinon.createSandbox();
 
 describe('lib/viewers/iframe/IFrameLoader', () => {
+    const iframe = new IFrameViewer({
+        file: {
+            id: '123',
+            extension: 'boxnote',
+            representations: {
+                entries: [
+                    {
+                        representation: 'ORIGINAL',
+                    },
+                ],
+            },
+        },
+    });
+
     afterEach(() => {
         sandbox.verifyAndRestore();
```

Comment:
This isn't directly related to your change, but we should remove these when we edit a test file, if possible. It's not actually being used in any tests in this case, either.